pycmr/granule.py

```diff
@@ -17,6 +17,12 @@ def __init__(self):
         self.params = {}
         self.options = {}
 
+    def _urlEncodeString(self, input):
```

Comment:
Similar to comment in #22, might be slicker to change this to `from urllib.parse import quote as urlEncodeString` if this method won't have any other custom functionality. Also worth considering following PEP8s guideline of underscores between words for method names.